docs/releases.md

```diff
@@ -2,23 +2,90 @@
-- **DietPi-Software** | **Steam** :octicons-arrow-right-16:  By automatically pulling in Box86 as dependency, Steam can now be installed on ARMv7 boards. It won't run perfectly stable yet and has limited features and game support, but we're optimistic that further improvements will address this in the future. For more details check the [documentation page](../software/gaming/#steam).
-- **DietPi-Software** | **mjpg-streamer** :octicons-arrow-right-16: Lightweight multi-source and multi-output plugin JPEG streamer, now available for install. By default your attached camera capture will be streamed to a custom HTTP port. When installed along with OctoPrint, both will be automatically configured to work together. The Raspberry Pi camera module support is enabled by default, when installed on Raspberry Pi.
+**[Docker Compose](../software/programming/#docker-compose)**
+
+Docker Compose is a tool to define and run multi-container Docker applications. It can now be installed through our software selection. [Docker](../software/programming/#docker) will be pulled-in automatically (as dependency).
+
+**What would Docker Compose bring to you ?**
+
+Whenever you have multiple containers :octicons-server-16: , there are many tasks you need to do: deploy and configure each individual container, and configure them to talk to each other as well. This will be tedious even with few containers.
+
+[Docker Compose](../software/programming/#docker-compose) lets you automate the deployment of multiple containers using an YAML file. This file enable you to configure your application’s services and create all the app’s services from this configuration.
+
+![docker compose](assets/images/dietpi-docker-compose.png){: style="width:500px" loading="lazy"}
+
+Find sample applications using Docker Compose and more details in the [documentation page](../software/programming/#docker-compose).
+
+**[Steam](../software/gaming/#steam)**  & **[Box86](../software/gaming/#box86)**
+
+[Steam](../software/gaming/#steam) can now be installed using DietPi (running on ARMv7 boards), with [Box86](../software/gaming/#box86) being installed automatically, as dependency.
+
+The Steam platform is one of the largest digital distribution platform for gaming. Still, on ARMv7 boards it has limited features and game support. Here are few sample games running with [Box86](../software/gaming/#box86):
+
+<iframe width="560" height="315" src="https://www.youtube-nocookie.com/embed/z-4aGNqZ724" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
```

Comment:
Let's use the enhanced YouTube iframe code I applied to the other pages:
```
<iframe src="https://www.youtube-nocookie.com/embed/z-4aGNqZ724?rel=0" frameborder="0" allow="fullscreen" width="560" height="315" loading="lazy"></iframe>
```
- `?rel=0` query string assures that only videos from the same channel are suggested after the video has ended. Let's limit YouTube's personalisation here to a minimum.
- Reduce feature policy permissions to a minimum and do not grant the ifram/YouTube access to various device sensors and autoplay, but fullscreen only.
- Add lazy loading, so the iframe is only loaded when it's close or inside the browsers viewport.